metrics_utility/automation_controller_billing/helpers.py

```diff
@@ -9,7 +9,11 @@
 from dateutil import parser
 from dateutil.relativedelta import relativedelta
 
-from metrics_utility.exceptions import UnparsableParameter
+from metrics_utility.exceptions import DateFormatError, UnparsableParameter
+
+
+DATE_REGEX_PATTERN = r'^\d{4}-\d{2}(-\d{2})?$'
+DAY_OR_MONTH_REGEX_PATTERN = r'.*(d|day|days|m|mo|month|months)$'
```

Comment:
```suggestion
DAY_OR_MONTH_REGEX_PATTERN = r'^\d+(d|day|days|m|mo|month|months)$'
```

(otherwise "Thursday" would be a valid value too)